ADotNet/Models/Pipelines/GithubPipelines/DotNets/Tasks/WorkloadUpdateTask.cs

```diff
@@ -0,0 +1,13 @@
+﻿// ---------------------------------------------------------------------------
+// Copyright (c) Hassan Habib & Shri Humrudha Jagathisun All rights reserved.
+// Licensed under the MIT License.
+// See License.txt in the project root for license information.
+// ---------------------------------------------------------------------------
+
+namespace ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks
+{
+    public class WorkloadUpdateTask : GithubTask
+    {
+        public override string Run { get; set; } = "dotnet workload update";
```

Comment:
Add descriptive documentation.

Reply:
@hassanhabib don't have an issue with this task itself, but but do we have some criteria on when to allow new tasks VS just using the generic task command in the job like below?

```cs
                    new GithubTask()
                    {
                        Name = "Update installed workloads",
                        Run = "dotnet workload update"
                    },
```